CHANGELOG.md

```diff
@@ -10,11 +10,12 @@
     - file.put_stream
     - file.get
     - file.get_stream
+  - snowflake.snowpark.functions
```

Comment:
This should be at line 17, `add_import` to `udf.register_from_file` are Session APIs.